.eslintrc

```diff
@@ -3,7 +3,10 @@
   "plugins": ["jest"],
   "rules": {
     "no-underscore-dangle": "off",
-    "no-use-before-define": "off"
+    "no-use-before-define": "off",
+    "no-continue": "off",
```

Comment:
Happy to change these back and refactor if you feel strongly about these linting rules.